packages/web/README.md

```diff
@@ -536,38 +536,31 @@ module.exports = {
 
 #### Source minification
 
-By default script sources are minified in production only, and using webpack's default of
-[uglifyjs-webpack-plugin](https://github.com/webpack-contrib/uglifyjs-webpack-plugin)
-(which internally uses `uglify-es`). To customise the options passed to `UglifyJsPlugin`
+By default script sources are minified in production only, using
+[terser-webpack-plugin](https://github.com/webpack-contrib/terser-webpack-plugin)
+(which replaces `uglifyjs-webpack-plugin`). To customise the options passed to `TerserPlugin`
 or even use a different minifier, override `optimization.minimizer`.
 
-Note: If switching to [babel-minify-webpack-plugin](https://github.com/webpack-contrib/babel-minify-webpack-plugin)
```

Comment:
I've intentionally removed this reference to babel-minify-webpack-plugin, since it's still virtually unmaintained/buggy/slow. If that changes in the future we can re-add.